### PR TITLE
HMOS TextViewOverflowing Library

### PR DESCRIPTION
**Pull request by vidyaakbar**

### CODE REVIEW

File: textviewoverflow/src/main/java/net/justinangel/textviewoverflow/TextViewOverflowing.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            // get the index of the last visible line
+            int lastLineIndex = getHeight() / lineHeight;
+            if (lastLineIndex > 0) {
+                setMaxTextLines(lastLineIndex);
+            }
+        }
+    }
+
+    private int calculateLineHeight() {
+        SimpleTextLayout simpleTextLayout = new SimpleTextLayout("SimpleTextLayout", mTextPaint, new Rect(), 0);
```
Comment (samuel-abhishek) [inline textviewoverflow/src/main/java/net/justinangel/textviewoverflow/TextViewOverflowing.java]: Avoid hardcoded strings. Use constants instead.

File: textviewoverflow/src/main/java/net/justinangel/textviewoverflow/TextViewOverflowing.java [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        return simpleTextLayout.getBottom(0) - simpleTextLayout.getTop(0);
+    }
+
+    String overflowText;
```
Comment (samuel-abhishek) [inline textviewoverflow/src/main/java/net/justinangel/textviewoverflow/TextViewOverflowing.java]: Declare the scope of this field. If its internal to the class make it private. Also declare all the fields before the functions for better code readability

**Review by samuel-abhishek — CHANGES_REQUESTED** [review]
Please check and handle the review comments. Also test it on SDK6 and update the build.gradle files in the project.

File: textviewoverflow/src/main/java/net/justinangel/textviewoverflow/TextViewOverflowing.java (hunk 1 above)
Comment (vidyaakbar) [reply textviewoverflow/src/main/java/net/justinangel/textviewoverflow/TextViewOverflowing.java]: Done

File: textviewoverflow/src/main/java/net/justinangel/textviewoverflow/TextViewOverflowing.java (hunk 2 above)
Comment (vidyaakbar) [reply textviewoverflow/src/main/java/net/justinangel/textviewoverflow/TextViewOverflowing.java]: Done

**Review by samuel-abhishek — APPROVED** [review]
LGTM

**Review by sharathks84 — APPROVED** [review]
OK Approved

**Review by Gleelakrishna — APPROVED** [review]
ok

**Pull request merged**